### DISCUSSION TITLE
How do I recognize that an issue was closed through a commit? (Fixes #123 Foo Blah)

### DISCUSSION THREAD

**Discussion by thadiun** [community/community#24889] [Apps, API and Webhooks]
<p>In a GitHub App, it is easy to listen to issues being closed. But how do I recognize that an issue is being closed because a commit in the style of <strong>Fixes <span class="hashtag">#123</span> Some Reason</strong> happened? The IssuesEvent does not contain any hints.</p>

**Comment by AndreaGriffiths11** [comment, marked as answer]
<p>Hi <a class="mention" href="/thadiun">@thadiun</a>,</p>
<p>Thanks for being here!  The event <code>closed</code> listed <a href="https://developer.github.com/v3/issues/timeline/#events" rel="nofollow noopener">here</a> The issue was closed by the <code>actor</code>. When the commit_id is present, it identifies the commit that closed the issue using “closes / fixes <span class="hashtag">#NN</span>” syntax.</p>
<p>I hope this helps!</p>
<p>Andrea</p>
